.tasks

```diff
@@ -14,7 +14,11 @@
 set -ex
 
 # Scan for all tests
-bots/tests-scan
+if [ -n "$AMQP_URL" ]; then
+    echo bots/run-queue --amqp "$AMQP_URL" --queue tasks
+else
+    echo bots/run-queue --amqp "amqp-cockpit.apps.ci.centos.org:443" --queue tasks
```

Comment:
How about calling good old tests-scan if `$AMQP_URL` is not set, and then set it in *both* clouds? This would avoid hardcoding the infra name in the cockpit git (which feels like a layering violation), would also make it easier to switch between AMQP and polling mode, and would also retain the ability to quickly spin up a few extra tasks containers somewhere to help out, without having to set up a webhook and AMQP for that.